pyproject.toml

```diff
@@ -32,7 +32,7 @@ dependencies = [
 ]
 
 [tool.setuptools]
-packages = ["cluster_utils", "cluster_utils.client", "cluster"]
+packages = ["cluster_utils", "cluster_utils.base", "cluster_utils.client", "cluster"]
```

Comment:
Is it necessary to declare the sub-packages here? Does it serve any particular purpose?

Reply:
Yes, it is necessary, otherwise they won't get installed.  It could probably be configured to auto-detect packages.  I'll look into this when adding the server sub-package.